apps/ios/Shared/Views/UserSettings/ScanSMPServer.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ScanSMPServer.swift
+//  SimpleX (iOS)
+//
+//  Created by Evgeny on 19/11/2022.
+//  Copyright © 2022 SimpleX Chat. All rights reserved.
+//
+
+import SwiftUI
+import SimpleXChat
+import CodeScanner
+
+struct ScanSMPServer: View {
+    @Environment(\.dismiss) var dismiss: DismissAction
+    @Binding var servers: [ServerCfg]
+    @State private var showAddressError = false
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            Text("Scan server QR code")
+                .font(.largeTitle)
+                .bold()
+                .padding(.vertical)
+            ZStack {
+                CodeScannerView(codeTypes: [.qr], completion: processQRCode)
+                    .aspectRatio(1, contentMode: .fit)
+                    .border(.gray)
+            }
+        }
```

Comment:
Maybe add some text below scanner similar to Connect views? The view feels unfinished atm. For example
```suggestion
        VStack(alignment: .leading) {
            Text("Scan server QR code")
                .font(.largeTitle)
                .bold()
                .padding(.vertical)
            ZStack {
                CodeScannerView(codeTypes: [.qr], completion: processQRCode)
                    .aspectRatio(1, contentMode: .fit)
                    .border(.gray)
            }
            Text("You can also add a server address manually.")
        }
```
(this is a bit redundant, maybe something else)

Reply:
That'll do for now :) I thought about it, but decided not to clutter it with the explanations - it's all pretty clear here.